nodes/parachain/src/chain_spec/polimec.rs

```diff
@@ -166,7 +166,7 @@ pub fn get_polkadot_base_chain_spec() -> Result<ChainSpec, String> {
 	))
 }
 
-pub fn get_rococo_base_chain_spec() -> Result<ChainSpec, String> {
+pub fn get_rococo_chain_spec() -> Result<ChainSpec, String> {
```

Comment:
The chain `name` in this case should be:


```suggestion
	Ok(ChainSpec::from_genesis(
		"Politest Rococo",
```